### ISSUE TITLE
ide (VSCode, phpstorm..etc) uri scheme implementation

### ISSUE THREAD

**Issue by xiaohuilam** [itsgoingd/clockwork-chrome#65]
please make these text link to IDE quick open scheme. or make this configurable at least.

![image](https://user-images.githubusercontent.com/6964962/44203241-eb473b80-a180-11e8-9847-050a2d6a86fb.png)

```
vscode://file/home/workspace/app/http/controllers/BastAuthController.php:58
```

Then we can quick go'to programming workspace and that's so great! 
 
 
explain:
```
vscode://file/{full-path-to-file}:{line-number}
```

**Comment by itsgoingd** [comment]
Hey, this is definitively a good idea. Adding to the todo list for the next release.

**Comment by itsgoingd** [comment]
This was released as part of the Clockwork 3.1.
